test/flow.tcl

```diff
@@ -191,6 +191,8 @@ write_verilog $verilog_file
 ################################################################
 # Global routing
 
+set_thread_count [exec getconf _NPROCESSORS_ONLN]
```

Comment:
Is there a reason not to put this at the start of the flow?  It won't make much difference today but it could in the future.

Reply:
The problems with mpl2 generating different results for different number of threads were resolved a while ago. It should be fine.